js/ui/gantt/ui.gantt.dialogs.js

```diff
@@ -0,0 +1,140 @@
+import Popup from "../popup";
+import dxForm from "../form";
+import $ from "../../core/renderer";
+
+export class GanttDialog {
+    constructor(owner, $element) {
+        this._popupInstance = owner._createComponent($element, Popup);
+
+        this.infoMap = {};
+        this.infoMap["TaskEdit"] = TaskEditDialogInfo;
+    }
+    _apply() {
+        const result = this._dialogInfo.getResult();
+        this._callback(result);
+        this.hide();
+    }
+
+    _clean() {
```

Comment:
Where is this method invoked?
Owner instance should manually call it, so `_` is unnecessary.